src/main/java/me/kodysimpson/simpapi/utility/TimeUtil.java

```diff
@@ -0,0 +1,181 @@
+            if (shortened)
+                minute = "m";
+            else
+                minute = (toMin == 1 ? "minute" : "minutes");
+            String second;
+            if (shortened)
+                second = "s";
+            else
+                second = (toSec == 1 ? "second" : "seconds");
+
+
+            return (toMin + (shortened ? "" : " ") + minute + (toMin == 1 && !secondNotZero ? "" : " ")) +
+                    (secondNotZero ? toSec + (shortened ? "" : " ") + second : "");
+        }
+
+        if (newTime <= 86040000) {
+
+            int toSec = (int) (newTime / 1000) % 60;
+            int toMin = (int) ((newTime / (1000 * 60)) % 60);
+            int toHour = (int) ((newTime / (1000 * 60 * 60)) % 24);
+
+            boolean hourNotZero = toHour != 0;
+            boolean minuteNotZero = toMin != 0;
+            boolean secondNotZero = toSec != 0;
+
+            String hour;
+            if (shortened)
+                hour = "h";
+            else
+                hour = (toHour == 1 ? "hour" : "hours");
+
+            String minute;
+            if (shortened)
+                minute = "m";
+            else
+                minute = (toMin == 1 ? "minute" : "minutes");
+            String second;
+            if (shortened)
+                second = "s";
+            else
+                second = (toSec == 1 ? "second" : "seconds");
+
+            return (hourNotZero ? toHour + (shortened ? "" : " ") + hour + " " : "") +
+                    (minuteNotZero ? toMin + (shortened ? "" : " ") + minute + " " : "") +
+                    (secondNotZero ? toSec + (shortened ? "" : " ") + second : "");
+        }
+
+        int toSec = (int) (newTime / 1000) % 60;
+        int toMin = (int) ((newTime / (1000 * 60)) % 60);
+        int toHour = (int) ((newTime / (1000 * 60 * 60)) % 24);
+        int toDays = (int) (newTime / (1000 * 60 * 60 * 24));
+
+        boolean dayNotZero = toDays != 0;
+        boolean hourNotZero = toHour != 0;
+        boolean minuteNotZero = toMin != 0;
+        boolean secondNotZero = toSec != 0;
+
+
+        String day;
+        if (shortened)
+            day = "d";
+        else
+            day = (toDays == 1 ? "day" : "days");
+
+        String hour;
+        if (shortened)
+            hour = "h";
+        else
+            hour = (toHour == 1 ? "hour" : "hours");
+
+        String minute;
+        if (shortened)
+            minute = "m";
+        else
+            minute = (toMin == 1 ? "minute" : "minutes");
+
+        String second;
+        if (shortened)
+            second = "s";
+        else
+            second = (toSec == 1 ? "second" : "seconds");
+
+        return (dayNotZero ? toDays + (shortened ? "" : " ") + day + " " : "") +
+                (hourNotZero ? toHour + (shortened ? "" : " ") + hour + " " : "") + (minuteNotZero ? toMin + (shortened ? "" : " ")
+                + minute + " " : "") + (secondNotZero ? toSec + (shortened ? "" : " ") + second : "");
```

Comment:
This is unmaintainable. Never do this. It is extremely unclear what the intended format is.
Use String#format, or if it's not possible to do it inline, use a StringBuilder and chain `append` calls.